tests/upgrades/test_errata.py

```diff
@@ -434,15 +399,14 @@ def test_post_scenario_generate_errata_with_previous_version_katello_agent_clien
         for errata in settings.repos.yum_9.errata:
             host.errata_apply(data={'errata_ids': [errata]})
 
-        for package in FAKE_9_YUM_UPDATED_PACKAGES:
-            install_or_update_package(client_hostname=client_container_id, package=package)
+        rhel_client.execute(f'yum -y install {" ".join(FAKE_9_YUM_UPDATED_PACKAGES)}')
+        # Do we need to install the packages on the client?
+        # The yum_9.errata(s) should already be installed
```

Comment:
Do we need to install the packages on the client? The yum_9.errata(s) should already be installed by the step above.